app/components/monitor/monitorCtrl.js

```diff
@@ -0,0 +1,37 @@
+(function () {
+
+  'use strict';
+
+  /**
+   * I can have stuff here and no other scope can see it!
+   * though you can't refer to any Angular stuff like $scope
+   * but for Objects or Arrays you could get clever with
+   */
+
+  function MonitorCtrl (MonitorService) {
+
```

Comment:
This is a good place to alias 'this' as something (e.g. 'var vm = this;')... once you begin to use response data from the MonitorService beyond logging it to the console, it'll serve you well. Anywhere you're attaching things to the 'this' of the controller, replace that with your aliased variable (e.g.: 

```
vm.monitor = function() {
  MonitorService.monitorServ(vm.url)
  .then(function\(response) {
    vm.myData = response.data;
  });
};
```